components/SplitPane/library/src/desktopMain/kotlin/org/jetbrains/compose/splitpane/DesktopSplitPane.kt

```diff
@@ -51,101 +50,91 @@ internal actual fun SplitPane(
             val secondMinSizePx = secondPlaceableMinimalSize.value * density
 
             with(splitPaneState) {
-                val constrainedMin = constraints.minByDirection(isHorizontal) + firstMinSizePx
-                val constrainedMax =
-                    (constraints.maxByDirection(isHorizontal).toFloat() - secondMinSizePx).let {
-                        if (it <= 0 || it <= constrainedMin) {
-                            constraints.maxByDirection(isHorizontal).toFloat()
-                        } else {
-                            it
-                        }
-                    }
+                val firstMeasurable = measurables[0]
```

Comment:
I guess [destructuring](https://kotlinlang.org/docs/reference/multi-declarations.html) will [work](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/component2.html) in this case